docs/source/services/timelock_service/manual-migration.rst

```diff
@@ -79,7 +80,8 @@ To verify that this step was completed correctly, you may curl the Timelock Serv
 
    .. code:: bash
 
-      curl -XPOST localhost:8080/test/timestamp/fresh-timestamp
+      curl -XPOST <protocol>://<host>:<port>/timelock/api/tl/ts/<namespace> \
+        --data '{"numTimestamps": 1}' -H "Authorization: Bearer q" -H "Content-Type: application/json"
```

Comment:
It'd probably be nice to have a callout/reminder that only the leader will respond, and other nodes will return a 308. Adding a `-i` to the curl was the only thing that tipped me off to the 308, otherwise there's just no output from non-leaders.

Reply:
Yep. I added the `-i` to display headers to the commands too.